script.module.win_inet_pton/addon.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.win_inet_pton" name="win-inet-pton" version="1.1.0" provider-name="hickeroar">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
```

Comment:
> WARN: For krypton it is advised to set xbmc.python version to 2.25.0

Reply:
@Rechi But there's no reason to artificially limit it to Krypton IMO. No problem to change this.